README.md

```diff
@@ -0,0 +1,39 @@
+# Fooocus-MindOfMatter-Edition
```

Comment:
@MindOfMatter just to make sure we're getting the baseline straight: please remove your readme changes from the PRs as Fooocus is not planned to be rebranded to Fooocus-MindOfMatter-Edition any time soon ^^
Thank you!